### PR TITLE
Added calls for singletons

### PR DESCRIPTION
**Pull request by nickhartjes**
I extended the SDK to support singletons

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,3 +1,2 @@
 indent_style = space
-indent_size = 2
-
+indent_size = 2
```
Comment (brunnolou) [inline .editorconfig]: Could you please not update this file? Since it's just removing the extra line.

File: .eslintignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 lib
 **/node_modules
+examples
```
Comment (brunnolou) [inline .eslintignore]: Same as above.

**Review by brunnolou — CHANGES_REQUESTED** [review]
Thanks for your PR! 💪

**Comment by nickhartjes** [comment]
Hi @brunnolou , i removed the changes on the .editorconfig .eslintignore

**Comment by nickhartjes** [comment]
Changes where done

**Review by brunnolou — APPROVED** [review]
Thanks! 👍

**Pull request merged**